test/letters/components/ErrorableSelect.unit.spec.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import ReactTestUtils from 'react-dom/test-utils';
+import SkinDeep from 'skin-deep';
+import chaiAsPromised from 'chai-as-promised';
+import chai, { expect } from 'chai';
+
+import ErrorableSelect from '../../../src/js/letters/components/ErrorableSelect';
+
+chai.use(chaiAsPromised);
+
+describe('<ErrorableSelect>', () => {
+  const testValue = '';
+  const options = [{ value: 1, label: 'first' }, { value: 2, label: 'second' }];
+
+  it('ensure value changes propagate', () => {
+    let errorableSelect;
+
+    const updatePromise = new Promise((resolve, _reject) => {
+      errorableSelect = ReactTestUtils.renderIntoDocument(
+        <ErrorableSelect label="test" options={options} value={testValue} onValueChange={(update) => { resolve(update); }}/>
+      );
+    });
+
+    const select = ReactTestUtils.findRenderedDOMComponentWithTag(errorableSelect, 'select');
+    select.value = '';
```

Comment:
It looks as though the initial value is set to an empty string on L12 / L20, then we're changing it to an empty string again here. So it seems as though we're testing that the change propagated but using the same pre-/post- change values. Might it be better to change one or the other?

And I think the same thing that applied to `ErrorableTextInput` applies here, right? We're only testing that `updatePromise` resolves with the new value (when `onValueChange` is called), but not sure if that's comprehensive (I seriously have no idea, this isn't a leading question!). Either way I don't think another test is strictly necessary in this PR, just wanted to bring this point up.

Reply:
Good catch on the update. Changed!

As far as the comprehensiveness, I think it's fine because we actually are simulating what happens when the select box is changed. We just have to trust that `ReactTestUtils.Simulate.change` does what it's supposed to and emulates it properly ;)